tests/integration/servicemesh/smmr_test.go

```diff
@@ -88,15 +99,24 @@ func TestSMMR(t *testing.T) {
 				verifyThatRouteExistsOrFail(t, namespaceGateway, gatewayName, "a.maistra.io")
 				verifyThatRouteExistsOrFail(t, namespaceGateway, gatewayName, "b.maistra.io")
 				verifyThatRouteIsMissingOrFail(t, namespaceGateway, gatewayName, "c.maistra.io")
+
+				if err := maistra.DisableIOR(t); err != nil {
+					t.Fatalf("failed to disable IOR: %s", err)
+				}
```

Comment:
Doesn't this need to be in a `defer` block so that it runs even if the ǜerifyThatRouteExistsOrFail()` function causes the test to terminate?